semantic-conventions/src/experimental_metrics.ts

```diff
@@ -682,7 +682,7 @@ export const METRIC_HW_POWER = 'hw.power' as const;
 /**
  * Operational status: `1` (true) or `0` (false) for each of the possible states
  * 
- * @note `hw.status` is currently specified as an *UpDownCounter* but would ideally be represented using a [*StateSet* as defined in OpenMetrics](https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md#stateset). This semantic convention will be updated once *StateSet* is specified in OpenTelemetry. This planned change is not expected to have any consequence on the way users query their timeseries backend to retrieve the values of `hw.status` over time.
+ * @note `hw.status` is currently specified as an *UpDownCounter* but would ideally be represented using a [*StateSet* as defined in OpenMetrics](https://github.com/prometheus/OpenMetrics/blob/v1.0.0/specification/OpenMetrics.md#stateset). This semantic convention will be updated once *StateSet* is specified in OpenTelemetry. This planned change is not expected to have any consequence on the way users query their timeseries backend to retrieve the values of `hw.status` over time.
```

Comment:
Ah, this is generated code, no need to update here as it'll be automatically updated once the semconv repo is released.

Reply:
reverted